tests/director/tests/application_templates_api_test.go

```diff
@@ -832,6 +832,56 @@ func TestDeleteApplicationTemplate(t *testing.T) {
 	saveExample(t, deleteApplicationTemplateRequest.Query(), "delete application template")
 }
 
+func TestDeleteApplicationTemplateWithCertSubjMapping(t *testing.T) {
+	// GIVEN
+	ctx := context.Background()
+	appTemplateName := createAppTemplateName("app-template")
+
+	tenantId := tenant.TestTenants.GetDefaultTenantID()
+
+	t.Log("Create application template")
```

Comment:
Add the application template name in the log. For example, `t.Logf("Create application template with name: %q", appTemplateName)`